.babelrc

```diff
@@ -1,19 +1,25 @@
 {
-  "plugins": [
-    ["transform-runtime", {
-      "polyfill": false,
-      "regenerator": true
-    }]
+  "presets": [
+    ["@babel/env", {
+      "loose": true,
+      "useBuiltIns": "usage",
+      "targets": {
+        "browsers": ["last 2 versions", "safari >= 7", "ie 9"],
```

Comment:
As already said, it is better to specify this in `package.json` browser list property.

Reply:
Sure, there are several of your suggestions I haven't implemented yet. I'm just trying to make progress on this project. I'm not a huge fan of having a giant pull request with so many changes, so I'd like to get this merged into master sooner rather than later. At that point I can tackle the smaller things.

Reply:
👍